Examples/JS/flow/PinType.js

```diff
@@ -0,0 +1,31 @@
+//
+//  PinType.js
+//  Autogenerated by plank
+//
+//  DO NOT EDIT - EDITS WILL BE OVERWRITTEN
+//  @generated
+//
+//  @flow
+//
+
+import type { PlankDate, PlankURI } from "./runtime.flow.js";
+import type BoardType from "./BoardType.js";
+import type ImageType from "./ImageType.js";
+import type UserType from "./UserType.js";
+
+export type PinType = {
+    +note?: string | null,
+    +media?: { [string]: string } | null,
```

Comment:
The contents of `media`, `counts`, etc should be write-only as well: `+media?: { +[string]: string }`.